src/Simulation/Simulators/OpenSystemsSimulator/DataModel/ChpDecomposition.cs

```diff
@@ -3,130 +3,129 @@
 
 #nullable enable
 
-using System;
-using System.Collections.Generic;
 using System.Text.Json;
 using System.Text.Json.Serialization;
 
-namespace Microsoft.Quantum.Experimental
-{
+namespace Microsoft.Quantum.Experimental;
 
-    public class ChpOperationConverter : JsonConverter<ChpOperation>
+public class ChpOperationConverter : JsonConverter<ChpOperation>
```

Comment:
I'm not sure if the file name `ChpDecomposition` relates to any identifier defined in this file. Up to you.

Reply:
Effectively, a decomposition is a list of operations, such that the file name was intended to reflect the group of classes that work together to represent that, but happy to rename if you prefer.

Reply:
Sounds reasonable, no need to rename. Thanks.